test/test_model.py

```diff
@@ -2199,40 +2362,43 @@ def _check_binding_data(self, binding_name: str, binding: ops.Binding):
         assert binding.network.bind_address == ipaddress.ip_address('192.0.2.2')
         assert binding.network.ingress_address == ipaddress.ip_address('192.0.2.2')
         # /32 and /128 CIDRs are valid one-address networks for IPv{4,6}Network types respectively.
-        assert binding.network.egress_subnets == [ipaddress.ip_network('192.0.2.2/32'),
-                                                  ipaddress.ip_network('192.0.3.0/24'),
-                                                  ipaddress.ip_network('dead:beef::/64'),
-                                                  ipaddress.ip_network('2001:db8::3/128')]
+        assert binding.network.egress_subnets == [
+            ipaddress.ip_network('192.0.2.2/32'),
+            ipaddress.ip_network('192.0.3.0/24'),
+            ipaddress.ip_network('dead:beef::/64'),
+            ipaddress.ip_network('2001:db8::3/128')
```

Comment:
Trailing commas (here and elsewhere). In future, hopefully the Ruff formatter does this for us automatically.

```suggestion
            ipaddress.ip_network('2001:db8::3/128'),
```